src/p2p/p2p_protocol_defs.h

```diff
@@ -467,6 +475,6 @@ namespace nodetool
 
 }
 
-BOOST_CLASS_VERSION(nodetool::peerlist_entry, 1)
+BOOST_CLASS_VERSION(nodetool::peerlist_entry, 2)
```

Comment:
This is an existing issue - the version declaration should be in the header where the serialization is declared. Otherwise serialization could be performed without this version being included.

Reply:
Fixed, moved to ` src/p2p/net_peerlist_boost_serialization.h` https://github.com/monero-project/monero/pull/5190/files#diff-48949eb87e7710c91d6683c2196398c7R44